### PR TITLE
Add Dell 7450 Sample PSAppDeploy

### PR DESCRIPTION
**Pull request by DenialP**

### CODE REVIEW

File: Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1 [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+		##* PRE-INSTALLATION
+		##*===============================================
+		[string]$installPhase = 'Pre-Installation'
+        Write-Log -Message "Checking status of BitLocker"
+        $bltemp = Get-BitLockerVolume
+        ForEach($x in $bltemp)
+        {
+            If($x.VolumeType -eq "OperatingSystem")
+            {
+                $blOS = $x
```
Comment (crossan007) [inline Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1:114]: What if there are multiple OS drives attached?

File: Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1 [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+		##*===============================================
+		##* INSTALLATION 
+		##*===============================================
+		[string]$installPhase = 'Installation'
+		
+		## Handle Zero-Config MSI Installations
+		If ($useDefaultMsi) { Execute-MSI -Action 'Install' -Path $defaultMsiFile }
+		
+		## <Perform Installation tasks here>
+		$results = Execute-Process -Path 'E7450A10.exe' -Parameters '/s /l=C:\Windows\Logs\Software\E7450A10_BIOS_Update.txt /p=PASSWORD' -PassThru
```
Comment (crossan007) [inline Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1]: should be E7450A10_BIOS_Update.log

File: Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1 [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+        #Execute-MSI -Action 'Install' -Path 'Media-Player_Windows_006_001_001.msi' -PassThru		
+
+		##*===============================================
+		##* POST-INSTALLATION
+		##*===============================================
+		[string]$installPhase = 'Post-Installation'
+		
+		## <Perform Post-Installation tasks here>
+        #If (Test-Path -Path "C:\Users\Public\Desktop\VLC media player.lnk") { Remove-File -Path "C:\Users\Public\Desktop\VLC media player.lnk" }
```
Comment (crossan007) [inline Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1]: probably don't need VLC and other misc

File: Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1 [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+	}
+	ElseIf ($deploymentType -ieq 'Uninstall')
+	{
+		##*===============================================
+		##* PRE-UNINSTALLATION
+		##*===============================================
+		[string]$installPhase = 'Pre-Uninstallation'
+		
+		## Show Welcome Message, close Internet Explorer with a 60 second countdown before automatically closing
+		#Show-InstallationWelcome -CloseApps 'vlc' -CloseAppsCountdown 60
```
Comment (crossan007) [inline Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1]: probably don't need VLC and other misc

File: Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1 [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+		
+		## Show Progress Message (with the default message)
+		Show-InstallationProgress
+		
+		## <Perform Pre-Uninstallation tasks here>
+		#Stop-Process -Name vlc -Force
```
Comment (crossan007) [inline Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1]: probably don't need VLC and other misc

File: Dell/BIOS/PSAppDeploy_Dell_E7450_(SCCM_SAMPLE).md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+If($Bios.SMBIOSBIOSVersion -ge $BiosMinimum)
+{
+    #requirements met, write something to output
+    Write-Host "BIOS is current"
+}
+#requirements not met, return nothing
+```
+
+#SCCM Return Code (Dell BIOS Returns are non-standard)
+Value: 2
```
Comment (crossan007) [inline Dell/BIOS/PSAppDeploy_Dell_E7450_(SCCM_SAMPLE).md]: This doesn't render nicely: https://github.com/bciu22/PowerShell-Modules/blob/DellBIOS/Dell/BIOS/PSAppDeploy_Dell_E7450_(SCCM_SAMPLE).md

See cheat sheat: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

File: Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1 (hunk 1 above)
Comment (DenialP) [inline Dell/BIOS/PSAppDeploy_Dell_E7450_(SAMPLE).ps1:114]: Gambling time!

**Pull request merged**